### ISSUE TITLE
Add notifications to imports

### ISSUE THREAD

**Issue by spylogsster** [brave/brave-browser#15176]
## Description 

Created from @bbondy comment https://github.com/brave/brave-core/pull/8422#issuecomment-814934266

Right clicking on a YouTube video (you need to right click twice because youtube tries to override the context menu), allows you to select import Video. But then it gives an error.

IPFS Companion also gives an error.
The error seems OK, but the way we deal with the error is that we don't do anything. IPFS companion does a notification like this for the error. We could do something similar unless you have a better suggestion.

![image](https://user-images.githubusercontent.com/2965009/113897885-018fe300-97d4-11eb-95ee-95c0706889be.png)

## Actual result:

- Nothing happens

## Expected result:
- Show notification about import error

## Reproduces how often: 

easy

## Version/Channel Information:

- Can you reproduce this issue with the current release? 
- Can you reproduce this issue with the beta channel? 
- Can you reproduce this issue with the nightly channel? 

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome?

**Comment by lidel** [comment]
I wrote some details about import failing on some websites in https://github.com/ipfs/ipfs-companion/issues/227 (mostly Firefox, but I feel a variant applies here too).

TLDR: In IPFS Companion it fails due to tracking protection (either regular CORS and other headers set by website, or browser itself behaving more strict for some reason, like in Firefox with protection enabled). 

We can't do much in a browser extension, but Brave should be able to side-steps those protections if the request was triggered by user interaction with native Import UI.

**Comment by stephendonner** [comment]
(Not yet setting the `QA Pass-Mac OS` label until I've been able to find and verify the `IPFS import partly completed` notification-message condition, from the screenshot above.  When I do, I'll return here and remove this text and amend the verification steps with the actual verification message.)

Steps:

(YouTube video)
1. loaded `https://www.youtube.com/watch?v=_28sorQWw_E&t=22s`
2. right-clicked on the video (twice)
3. chose `Import to IPFS > Selected video`
4. got the import-error message notification below

(MP4 video)
1. loaded `https://www.learningcontainer.com/mp4-sample-video-files-download/`
2. right-clicked on the 1st video
3. chose `Import to IPFS > Selected video`
4. got the `IPFS Import Completed` message notification below
5. confirmed I was able to share the video via link: `https://bafybeics24zwxekzwol2brpsubz6es6awjn2eolvpa3jlafqgt4qxvbm3q.ipfs.dweb.link/?filename=sample-mp4-file.mp4`

example | example | example
---------|---------|---------
<img width="1680" alt="Screen Shot 2021-04-19 at 6 30 17 PM" src="https://user-images.githubusercontent.com/387249/115324526-e364a400-a13e-11eb-9b86-e2df95c05d97.png"> | <img width="1680" alt="Screen Shot 2021-04-19 at 6 28 35 PM" src="https://user-images.githubusercontent.com/387249/115324078-19edef00-a13e-11eb-841a-d21367e4e9a9.png">

**Comment by stephendonner** [comment]
@spylogsster @lidel do either of you happen to have a handy use-case for `IPFS import partly completed`?  Thanks!

**Comment by spylogsster** [comment]
@stephendonner try to import same link twice, next imports will be partially completed because same link already exists in the target directory
